lightspark/examples/example.rs

```diff
@@ -32,6 +32,7 @@ async fn main() {
     let api_token = std::env::var("LIGHTSPARK_API_TOKEN_CLIENT_SECRET").unwrap();
 
     let node_password = std::env::var("LIGHTSPARK_TEST_NODE_PASSWORD").unwrap();
+    let base_url = std::env::var("LIGHTSPARK_API_ENDPOINT").unwrap();
```

Comment:
The other SDKs examples use `LIGHTSPARK_EXAMPLE_BASE_URL`, maybe this should be consistent?